src/Service/ImageCropperResizer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Service;
+
+use App\Base64EncodedFileTransformers\Base64EncodedFile;
+use App\Base64EncodedFileTransformers\UploadedBase64EncodedFile;
+use Imagick;
+use ImagickException;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class ImageCropperResizer
+{
+    public function avatarCropResize(UploadedFile $userAvatar, int $size): UploadedBase64EncodedFile
```

Comment:
If the class name is general, use general variables. Not `avatarCropResize()` but `cropAndResize()`. Not `$userAvatar` but `$uploadedFile`.

Reply:
Also maybe I need a diferent width than height?